README.md

```diff
@@ -1,52 +0,0 @@
-# People Fall Detection
-
-_This repo is the new home of the fall detection model used in Ambianic Edge. We are preparing to move the code over from the [ambianic-edge repo](https://github.com/ambianic/ambianic-edge/blob/master/src/ambianic/pipeline/ai/fall_detect.py) and make it available as a standalone library._
-
-# TODO
-
-- [ ] Standalone Python ML library for people fall detection based on Tensorflow and [PoseNet 2.0](https://github.com/tensorflow/tfjs-models/tree/master/posenet).
-- [ ] Python source code and wheel package published on PyPi 
-- [ ] Jupyter Notebook to interactively test and experiement with the model
-- [ ] CI & test suite
-- [ ] Training and testing data sets
-- [ ] Third party ML models used as building blocks
```

Comment:
@bhavikapanara this README needs to have sections on basic usage for library API, jupyter notebook usage and CLI usage.

Reply:
added